dashboard/app/helpers/survey_results_helper.rb

```diff
@@ -1,38 +1,45 @@
 module SurveyResultsHelper
-  def show_survey?
-    # rubocop:disable Lint/UnreachableCode
+  def show_diversity_survey?
```

Comment:
Sure.  Between this change and the earlier work in https://github.com/code-dot-org/code-dot-org/pull/12583 the goal has certainly been to make it trivial to turn these surveys on and off.